metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -145,6 +145,26 @@ public void run() {
     }
 
     /**
+     * Starts the reporter polling at the given period.
+     *
+     * @param period       the amount of time between polls
+     * @param initialDelay the time to delay the first execution
+     * @param unit         the unit for {@code period}
+     */
+    public void start(long period, long initialDelay, TimeUnit unit) {
```

Comment:
I think it would be better to change the order of parameters, so `initialDelay` comes first, to maintain the same order as in the `scheduleAtFixedRate` method.